pkg/gcv/validator_test.go

```diff
@@ -233,7 +233,11 @@ func testOptions() ([]string, string) {
 
 var storageAssetNoLoggingJSON = `{
   "name": "//storage.googleapis.com/my-storage-bucket",
-  "ancestry_path": "organization/1/folder/2/project/3",
+  "ancestors": [
```

Comment:
should we add another case rather than modifying this to show it should work for both cases where  there is no ancestry path or when there is no ancestors?